### ISSUE TITLE
"Wrap in widget" doesn't appear in right-click

### ISSUE THREAD

**Issue by sethladd** [flutter/flutter-intellij#1030]
Did we consider increasing the findability by putting "wrap in widget" and other features like that into the right-click menu ?

When I select a widget constructor in my build(), I right-clicked and didn't find options to help with managing that.

I found https://flutter.io/intellij-ide/ which tells me to Alt-Click (or, on a mac, option-click) to find the cool features.

Thanks for considering right-click :)

**Comment by devoncarew** [comment]
@messick, what's the feasibility of also exposing targeted flutter specific code actions via the context menu?

**Comment by alexander-doroshko** [comment]
Normally we do not include such actions in right-click menu. There's an advertising bulb already, isn't there?
![image](https://cloud.githubusercontent.com/assets/4499789/26152792/2c03e240-3b11-11e7-9402-ef6ee180e4a7.png)

![image](https://cloud.githubusercontent.com/assets/4499789/26152817/49ddf9ea-3b11-11e7-8cac-dc4d9a811353.png)

**Comment by stevemessick** [comment]
@sethladd Yes, we did consider it. In the end we went with the UI paradigm, rather than do something unusual (in the IntelliJ context, that is).

@devoncarew I think the feasibility is pretty low, given Alex's response. We can of course discuss it further, and I have some ideas.

@alexander-doroshko Are you going to bring some of those shirts to the next Dart summit? :)

**Comment by alexander-doroshko** [comment]
> Are you going to bring some of those shirts to the next Dart summit? :)

AFAIR we were giving them away at Devoxx back in 2013 where we met first time :) (that's why an old logo is there).

**Comment by pq** [comment]
FYI @jwren

**Comment by harshrocks** [comment]
Consider updating the flutter and dart plugins and it should work wonders
